tests/processor/test_expired_data_remover.py

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Test the ExpiredDataRemover object."""
+
+from unittest.mock import patch
+from datetime import datetime
+
+from masu.external.date_accessor import DateAccessor
+from masu.processor.expired_data_remover import ExpiredDataRemover
+from tests import MasuTestCase
+
+
+class ExpiredDataRemoverTest(MasuTestCase):
+    """Test Cases for the ExpiredDataRemover object."""
+
+    def test_initializer(self):
+        """Test to init"""
+        remover = ExpiredDataRemover('testcustomer')
+        self.assertEqual(remover._months_to_keep, 3)
+        self.assertIsInstance(remover._expiration_date, datetime)
+
+    def test_calculate_expiration_date(self):
+        """Test that the expiration date is correctly calculated."""
+        date_matrix = [{'current_date':    datetime(year=2018, month=7, day=1), 
```

Comment:
It's a little unclear what cases this text matrix covers. I'd like to see a code comment describing the intent so that we don't have to guess at it.

Reply:
This is all about taking sure the calculated expiration date is correct given the current date.  I'll add a comment on next round to clarify.